api-tests/features/cimit-journey.feature

```diff
@@ -0,0 +1,100 @@
+      When I submit an 'appTriage' event
+      Then I get a 'dcmaw' CRI response
+      When I submit 'kenneth-passport-valid' details to the CRI stub with 'NEEDS-ENHANCED-VERIFICATION' CI to mitigate
+      Then I get a 'page-dcmaw-success' page response
+      When I submit a 'next' event
+      Then I get an 'address' CRI response
+      When I submit 'kenneth-current' details to the CRI stub
+      Then I get a 'fraud' CRI response
+      When I submit 'kenneth-score-2' details to the CRI stub
+      Then I get a 'page-ipv-success' page response
+      When I submit a 'next' event
+      Then I get an OAuth response
+      When I use the OAuth response to get my identity
+      Then I get a 'P2' identity
+
+    Scenario: CiMit - Enhanced verification mitigation via F2F
+      When I submit a 'f2f' event
+      Then I get a 'f2f' CRI response
+      When I submit 'kenneth-driving-permit-valid' details to the async CRI stub with 'NEEDS-ENHANCED-VERIFICATION' CI to mitigate
+      Then I get a 'page-face-to-face-handoff' page response
+
+      # Return journey
+      When I start a new 'medium-confidence' journey and return to a 'page-ipv-reuse' page response
+      When I submit a 'next' event
+      Then I get an OAuth response
+      When I use the OAuth response to get my identity
+      Then I get a 'P2' identity
+
+
+  Rule: CiMit - Alternate doc mitigation
+    Scenario Outline: CiMit - Alternate doc mitigation via passport or DL
+      Given I start a new 'medium-confidence' journey
+      Then I get a 'page-ipv-identity-document-start' page response
+      When I submit an 'appTriage' event
+      Then I get a 'dcmaw' CRI response
+      When I get an 'access_denied' OAuth error from the CRI stub
+      Then I get a 'page-multiple-doc-check' page response
+      When I submit an <initialAlternateCri> event
+      Then I get a <initialAlternateCri> CRI response
+      When I submit <initialInvalidDoc> details to the CRI stub
+      Then I get a <noMatchPage> page response
+      When I submit a 'next' event
+      Then I get a <mitigatingCri> CRI response
+      When I submit <mitigatingDoc> details to the CRI stub with 'NEEDS-ALTERNATE-DOC' CI to mitigate
+      Then I get an 'address' CRI response
+      When I submit 'kenneth-current' details to the CRI stub
+      Then I get a 'fraud' CRI response
+      When I submit 'kenneth-score-2' details to the CRI stub
+      Then I get a 'page-pre-experian-kbv-transition' page response
+      When I submit a 'next' event
+      Then I get a 'kbv' CRI response
+      When I submit 'kenneth-score-2' details to the CRI stub
+      Then I get a 'page-ipv-success' page response
+      When I submit a 'next' event
+      Then I get an OAuth response
+      When I use the OAuth response to get my identity
+      Then I get a 'P2' identity
+
+      Examples:
+        | initialAlternateCri | initialInvalidDoc | noMatchPage | mitigatingCri | mitigatingDoc |
```

Comment:
I think `initialAlternateCri` is probably just `initialCri` or similar (it's not alternate yet!)